vessim/__init__.py

```diff
@@ -110,11 +113,54 @@ def __init__(
 
         self._fill_method = fill_method
 
+    @classmethod
+    def from_dataset(
+        cls,
+        dataset: str,
+        data_dir: Optional[str] = None,
+        scale: float = 1.0,
+        offset: float = 0.0,
+        start_time: Optional[DatetimeLike] = None,
+        use_forecasts: bool = True,
+        **kwargs
+    ):
+        path, files = download_dataset(dataset, data_dir)
+
+        # Currently specific for Solcast, needs change if more datasets are introduced
+        def _solcast_transform(df):
+            df = df.unstack(level=0)
+            df.columns = [col[1] for col in df.columns]
+            return df
```

Comment:
yes what I mean is, since we are the ones who supply the datasets, why not provide the CSV files in the correct format in the first place? This way people can reference them to understand how the CSV files are supposed to look like